### PR TITLE
update exokit.sh path

### PR DESCRIPTION
**Pull request by chrislatorres**
Updating paths to support changes to file locations.

### CODE REVIEW

**Comment by chrislatorres** [comment]
at https://github.com/webmixedreality/exokit/blob/master/tests/unit/__setup.test.js#L10

in exokit/tests/unit running node ../../src/native-bindings.js has the error for libopenvr_api.so
but in exokit/tests
running node ../src/native-bindings.js runs fine

File: src/native-bindings.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const path = require('path');
-const bindings = require(path.join(__dirname, '../', 'build', 'Release', 'exokit.node'));
+const bindings = require(path.join(__dirname, '..', 'build', 'Release', 'exokit.node'));
```
Comment (modulesio) [inline src/native-bindings.js]: Why this change? Does it do anything?

Comment (chrislatorres) [reply src/native-bindings.js]: It doesn't do anything technically, it makes it less confusing without the slash.
